task01/script.js

```diff
@@ -0,0 +1,65 @@
+/*
+    Tucker Kent
+    script.js
+    19SP_INFO_2134_WW Online - JavaScript II
+    Thoendel
+    8 April 2020
+*/
+
+window.addEventListener("load", (event) => { //window event listener set to run this code on load event
+
+    const firstName = document.getElementById("firstName"); //setting firstName input field location to constant
+    const lastName = document.getElementById("lastName");//setting lastName input field location to constant
+    const phoneNum = document.getElementById("telephoneNumber");//setting telephoneNumber input field location to constant
+
+    document.addEventListener("submit", (e) => { //event listener on submit event
+        let firstBool = false; //boolean variable for first name validation
+        let lastBool = false; //boolean variable for last name validation
+        let phoneBool = false; //boolean variable for phone number validation
+        if(firstName.value === ""){ //checking value of first name -- if blank
+            e.preventDefault(); // preventing submit action
+            firstBool = true; //setting boolean to true for error message
+        };
+        if(lastName.value === ""){ // checking value of last name -- if blank
+            e.preventDefault(); // preventing submit action
+            lastBool = true; //setting boolean to true for error message
+        };
+        if (checkNum(phoneNum.value) === false){
+            e.preventDefault(); // preventing submit action
+            phoneBool = true; //setting boolean to true for error message
+        };
+        let errorMessage = errorGenerator(firstBool, lastBool, phoneBool); //generating error message with errorGenerator function
+        if(errorMessage != ""){ //checks if the error message is blank
+            alert(errorMessage); //alerts user of the errors
+        };
+        
+    });
+
+    function checkNum(phoneNumber) { //function to check if the phone number is valid against the RegExp 
+        const pattern = new RegExp(/^(?:\([2-9]\d{2}\)\ ?|[2-9]\d{2}(?:\-?|\ ?))[2-9]\d{2}[- ]?\d{4}$/); //this is the closest I could find on the site -- there is an error
```

Comment:
I didn't encounter the error. I would want both parentheses - or none. Your pattern matched this.